src/platform_impl/ios/view.rs

```diff
@@ -197,6 +296,43 @@ impl WinitView {
         this
     }
 
+    pub(crate) fn recognize_pinch_gesture(&self, should_recognize: bool) {
+        if should_recognize && self.ivars().pinch_gesture_recognizer.borrow().is_none() {
+            let pinch = UIPinchGestureRecognizer::init_with_target(self, sel!(pinchGesture:));
+            self.addGestureRecognizer(pinch.as_super());
+            self.ivars().pinch_gesture_recognizer.replace(Some(pinch));
+        } else if let Some(recognizer) = self.ivars().pinch_gesture_recognizer.take() {
+            self.removeGestureRecognizer(&recognizer);
+        }
```

Comment:
I think the logic here is wrong? If I have `should_recognize = true` and a recognizer is already added, then this will remove it.